packages/atlas/src/components/NumberFormat/NumberFormat.tsx

```diff
@@ -67,6 +68,10 @@ export const NumberFormat = forwardRef<HTMLHeadingElement, NumberFormatProps>(
       </StyledText>
     )
 
+    if (format === 'dollar' && !JOY_PRICE_SERVICE_URL) {
+      return null
+    }
```

Comment:
Does this needs to rely on `JOY_PRICE_SERVICE_URL`? I would expect the `convertHapiToUSD` to return null-ish value if the price is not available so we wouldn't need to check here for the URL